Cargo.toml

```diff
@@ -163,6 +163,7 @@ federated-server = { path = "crates/federated-server" }
 gateway-config = { path = "crates/gateway-config" }
 rolling-logger = { path = "crates/rolling-logger" }
 grafbase-sdk = { path = "crates/grafbase-sdk" }
+extension = { path = "crates/extension" }
```

Comment:
You can and should use this from the workspace now.

Reply:
it is the workspace, not sure what you mean